markdown/developers/sync.md

```diff
@@ -371,21 +371,23 @@ If so, the easiest solution is to start your `TEMPLATE` branch from scratch.
 * Clone the main nf-core pipeline repository to your local machine (not your development fork)
 
   ```bash
-  git clone <git URL>
+  git clone git@github.com:nf-core/<PIPELINE>.git
```

Comment:
Doesn't this assume someone is using ssh keys though?

Reply:
```suggestion
  git clone https://github.com/nf-core/<PIPELINE>
```

Reply:
Do people not use ssh keys?

Reply:
@Zethson - missing the `.git`

```suggestion
  git clone https://github.com/nf-core/<PIPELINE>.git
```

Reply:
@ewels very few do from my experience. The default is certainly not.

The `.git` is not required. you can clone without it.

Reply:
I've seen mixed, which is why originally I think I didn't specify the exact clone mechanism.

Reply:
I would say that the people that are somewhat new to git/Github will use https and not clone using their SSH keys. The people however that do use their SSH keys will know how to adapt the URL to their needs, but this does not hold the other way around (more or less).